File: module06/module025/module1/src/main/java/com/android/example/module06_module025_module1/ClassAAA.java
package com.android.example.module06_module025_module1;

public class ClassAAA {
	private 	com.android.example.module06_module025_module2.ClassAAC instance_var_1_0 = new com.android.example.module06_module025_module2.ClassAAC();
	private 	com.android.example.module06_module109_module1.ClassAAG instance_var_1_1 = new com.android.example.module06_module109_module1.ClassAAG();
	private 	com.android.example.module06_module025_module2.ClassAAA instance_var_1_2 = new com.android.example.module06_module025_module2.ClassAAA();
	private 	com.android.example.module06_module025_module2.ClassAAI instance_var_1_3 = new com.android.example.module06_module025_module2.ClassAAI();
	private 	com.android.example.module06_module109_module1.ClassAAC instance_var_1_4 = new com.android.example.module06_module109_module1.ClassAAC();

	public void method0(
		com.android.example.module06_module109_module1.ClassAAH param0,
		com.android.example.module06_module109_module1.ClassAAC param1) throws Throwable {
	}

	public void method1(
		com.android.example.module06_module025_module2.ClassAAG param0) throws Throwable {
		param0.method2(new io.reactivex.disposables.SerialDisposable(), new io.reactivex.subscribers.TestSubscriber());

		if (new java.lang.Object().equals(new java.lang.Object())) {
			for (int iAb = 0; iAb < 0; iAb++) {
				param0.method2(new io.reactivex.disposables.SerialDisposable(), new io.reactivex.subscribers.TestSubscriber());

				param0.method1(new io.reactivex.subscribers.TestSubscriber());

				io.reactivex.subscribers.TestSubscriber<Object> local_var_4_0 = new io.reactivex.subscribers.TestSubscriber();
				local_var_4_0.assertNotSubscribed();

			}
		}
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_1 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_1.assertNotSubscribed();

	}

	public void method2(
		com.android.example.module06_module109_module1.ClassAAD param0,
		com.android.example.module06_module109_module1.ClassAAB param1,
		com.android.example.module06_module109_module1.ClassAAB param2) throws Throwable {
		io.reactivex.internal.disposables.ListCompositeDisposable local_var_2_3 = new io.reactivex.internal.disposables.ListCompositeDisposable();
		local_var_2_3.clear();

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_4 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_4.requestMore(-1872260081l);

		io.reactivex.internal.disposables.ListCompositeDisposable local_var_2_5 = new io.reactivex.internal.disposables.ListCompositeDisposable();
		local_var_2_5.isDisposed();

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_6 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_6.cancel();

	}
}

File: module06/module284/module3/src/main/java/com/android/example/module06_module284_module3/ClassAAA.kt
package com.android.example.module06_module284_module3

@Suppress("UNUSED_PARAMETER")
class ClassAAA {
	private 	val instance_var_1_0: com.android.example.module06_module284_module5.ClassAAI = com.android.example.module06_module284_module5.ClassAAI()
	private 	val instance_var_1_1: com.android.example.module06_module284_module5.ClassAAJ = com.android.example.module06_module284_module5.ClassAAJ()
	private 	val instance_var_1_2: com.android.example.module06_module284_module2.ClassAAE = com.android.example.module06_module284_module2.ClassAAE()

	fun method0(
		param0: com.android.example.module06_module284_module5.ClassAAC
	) {
		val local_var_2_1: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_1.cancel()

		listOf("1", "2", "3").forEach {
			for (iAb in 0..0) {
				val local_var_4_0: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
				local_var_4_0.isCancelled()

				val local_var_4_1: io.reactivex.internal.disposables.ListCompositeDisposable = io.reactivex.internal.disposables.ListCompositeDisposable()
				local_var_4_1.dispose()

			}
			param0.method0(io.reactivex.subscribers.TestSubscriber(), io.reactivex.subscribers.TestSubscriber(), io.reactivex.internal.schedulers.SingleScheduler(), io.reactivex.subscribers.TestSubscriber())

		}
		val local_var_2_2: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_2.onError(kotlin.Throwable())

		val local_var_2_3: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_3.requestMore(7952843864257707369)

	}

	fun method1(
		param0: com.android.example.module06_module284_module2.ClassAAI,
		param1: com.android.example.module06_module284_module5.ClassAAE,
		param2: com.android.example.module06_module284_module5.ClassAAB,
		param3: com.android.example.module06_module284_module5.ClassAAB
	) {
	}

	fun method2(
		param0: com.android.example.module06_module284_module2.ClassAAD,
		param1: com.android.example.module06_module284_module5.ClassAAF,
		param2: com.android.example.module06_module284_module5.ClassAAG,
		param3: com.android.example.module06_module284_module2.ClassAAI
	) {
		val local_var_2_4: io.reactivex.internal.util.LinkedArrayList = io.reactivex.internal.util.LinkedArrayList(81)
		local_var_2_4.toString()

		val local_var_2_5: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_5.isDisposed()

		val local_var_2_6: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_6.hasSubscription()

	}

	fun method3(
		param0: com.android.example.module06_module284_module5.ClassAAH
	) {
		val local_var_2_1: io.reactivex.internal.schedulers.ComputationScheduler = io.reactivex.internal.schedulers.ComputationScheduler()
		local_var_2_1.start()

		param0.method1(io.reactivex.subscribers.TestSubscriber(), io.reactivex.internal.disposables.ListCompositeDisposable(), io.reactivex.subscribers.TestSubscriber(), io.reactivex.subscribers.TestSubscriber())

		param0.method0(io.reactivex.subscribers.TestSubscriber())

	}
}
